src/target_info.rs

```diff
@@ -5,10 +5,21 @@ pub const RISCV_ARCH_MAPPING: &[(&str, &str)] = &[
     ("riscv32gc", "riscv32"),
     ("riscv32i", "riscv32"),
     ("riscv32im", "riscv32"),
-    ("riscv32ima", "riscv32"),
```

Comment:
Is this deletion accidental?

Reply:
At first I thought this array was out of date but now realised I've been running this tool with stable toolchain...